scripts/run-tests-local.ps1

```
@@ -0,0 +1,53 @@
+} catch {
+    docker-compose up -d | Out-Null
+}
+
+Write-Host "Waiting for Postgres to be ready..."
+$max = 120
+$i = 0
+while ($i -lt $max) {
+    try {
+        docker exec acbu-postgres pg_isready -U ${env:POSTGRES_USER -or 'acbu'} > $null 2>&1
+        if ($LASTEXITCODE -eq 0) { break }
+    } catch { }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)